root/static/scripts/release-editor/trackParser.js

```diff
@@ -33,16 +33,22 @@ releaseEditor.trackParser = {
     trackNumber: /^(?:M[\.\-])?([０-９0-9]+(?:-[０-９0-9]+)?)(?:\.|．|\s?-|:|：|;|,|，|$)?/,
     vinylNumber: /^([０-９0-9a-z]+)(?:\/[０-９0-9a-z]+)?(?:\.|．|\s?-|:|：|;|,|，|$)?/i,
 
-    trackTime: /\(?((?:[0-9０-９]+[：，．':,.])?[0-9０-９\?]+[：，．':,.][0-5０-５\?][0-9０-９\?])\)?$/,
+    trackTime: new RegExp("\\(?((?:[0-9０-９]+[：，．':,.])?[0-9０-９\\?]+[：，．'\\+:,.][0-5０-５\\?][0-9０-９\\?])\\)?$"),
```

Comment:
Not sure why this was changed to `new RegExp`, since the pattern still looks to be static.

Reply:
I changed this when this originally was for a tracktime delimiter, but forgot to change it back D: